lib/routes/account.js

```diff
@@ -516,9 +528,15 @@ module.exports = function (
         }
 
         function sendNewDeviceLoginNotification() {
-          // New device notification emails should only be sent for requesting keys and
-          // not performing a sign-in confirmation.
-          var shouldSendNewDeviceLoginEmail = config.newLoginNotificationEnabled && requestHelper.wantsKeys(request) && !doSigninConfirmation
+          // New device notification emails should only be sent when requesting keys.
+          // They're not sent if performing a sign-in confirmation
+          // (in which case you get the sign-in confirmation email)
+          // or if the account is unverified (in which case
+          // content-server triggers a resend of the account verification email)
+          var shouldSendNewDeviceLoginEmail = config.newLoginNotificationEnabled
+          shouldSendNewDeviceLoginEmail &= requestHelper.wantsKeys(request)
```

Comment:
I find the bitwise anding here a bit confusing, any advantage of doing this over using normal chained boolean operators?